renovate.json

```diff
@@ -1,11 +1,12 @@
 {
-  "enabled": false,
+  "enabled": true,
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": ["config:base"],
   "prConcurrentLimit": 10,
   "branchConcurrentLimit": 0,
   "branchPrefix": "renovate-",
   "prHourlyLimit": 1,
+  "rebaseWhen": "never",
   "rangeStrategy": "bump",
   "prCreation": "not-pending",
```

Comment:
:thinking: 
>Avoid setting rebaseWhen=never and then also setting prCreation=not-pending as this can prevent creation of PRs.

https://docs.renovatebot.com/configuration-options/#rebasewhen